src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margins and paddings using * selector. Specify the exact tags you want to reset instead